packages/events/src/util.ts

```diff
@@ -21,60 +22,56 @@ import { getHubUrl } from "@esri/hub-domains";
  * Fetch the events service associated with a Hub Site.
  * @param orgId - Identifier of the ArcGIS Online Organization
```

Comment:
to differentiate between these new methods, i think our best option is to be explicit in the TSDoc code snippet about what is returned

```
* request("http://custom.maps.arcgis.com/sharing/rest/portals/self")
*   .then(response => {
*     getEventServiceUrl(response.id)
*       .then(url)
*         //  "https://hub.arcgis.com/..." or "https://services.arcgis.com/..."
*   })
```